core/src/main/java/fj/Bottom.java

```diff
@@ -52,10 +52,8 @@ public static Error error(final String s) {
    * @return A function that throws an error using the given message, ignoring its argument.
    */
   public static <A, B> F<A, B> errorF(final String s) {
-    return new F<A, B>() {
-      public B f(final A a) {
-        throw new Error(s);
-      }
+    return a -> {
+      throw new Error(s);
```

Comment:
@mperry particularly here (because of throw) brackets are necessary. Parser fails in absence of them.